CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ This release adds an embedded SQLite database for storing metadata required by t
 1. [21786](https://github.com/influxdata/influxdb/pull/21786): Ported the `influxd inspect deletetsm` command from 1.x.
 1. [21802](https://github.com/influxdata/influxdb/pull/21802): Removed unused `chronograf-migator` package & chronograf API service, and updated various "chronograf" references.
 1. [21828](https://github.com/influxdata/influxdb/pull/21828): Added the command `influx inspect verify-wal`.
+1. [21887](https://github.com/influxdata/influxdb/pull/21887): chore(ae): add more logging
```

Comment:
Since there's no user-visible impact of the change (that I can tell), I think we can leave it out of the log

Reply:
Reverted.